.github/workflows/post-release-update_bpmn_visualization_version_in_R_repo.yml

```diff
@@ -15,22 +15,22 @@ jobs:
       VERSION: ${{ github.event.client_payload.version || github.event.inputs.version }}
     steps:
       - uses: actions/checkout@v3
-      - name: Get old BPMN Visualization version
+      - name: Get the old version of bpmn-visualization
         id: lookupOldBPMNVisuVersion
         uses: mikefarah/yq@v4.30.6
         with:
           cmd: yq '.dependencies[] | select(.name == "bpmn-visualization") | .version' inst/htmlwidgets/bpmnVisualization.yaml
       - run: echo "OLD_VERSION=${{ steps.lookupOldBPMNVisuVersion.outputs.result }}" >> $GITHUB_ENV
-      - name: Delete old BPMN Visualization version
+      - name: Delete the old bpmn-visualization
         run: rm inst/htmlwidgets/lib/bpmn-visualization/bpmn-visualization.min.js
-      - name: Download BPMN Visualization ${{ env.VERSION }}
+      - name: Download bpmn-visualization@${{ env.VERSION }}
         uses: carlosperate/download-file-action@v2
         with:
           # Use a hard coded version as the release version in this repository doesn't follow bpmn-visualization release versions
           # This ensures we can download the js file
           file-url: 'https://cdn.jsdelivr.net/npm/bpmn-visualization@0.23.1/dist/bpmn-visualization.min.js'
           location: 'inst/htmlwidgets/lib/bpmn-visualization'
-      - name: Update BPMN Visualization version in YAML file
+      - name: Update the bpmn-visualization version in the YAML file
         run: |
           sed -i -E 's/version: ".*"/version: "${{ env.VERSION }}"/' inst/htmlwidgets/bpmnVisualization.yaml
```

Comment:
⚠️ this line and the related file name should be updated, `bpmnVisualization` --> `bpmnVisualizationR`

Reply:
This will be handled with #217